umbraco/Umbraco-CMS
# RTE data-udi and image paths

fixes: https://github.com/umbraco/Umbraco-CMS/issues/5123

Updated RTE value converter to remove `data-udi` attributes from `a` and `img` tags when rendering.

Updated RTE property editor to strip the media path from the saved `img` tag if a `data-udi` attribute exists so we don't store potential stale paths.

## What to test

Add an image (or multiple) to the RTE 
- Check that the saved value in the DB only contains the querystring part of the path
- Check that the rendered `img` tag on the frontend has the correct path and doesn't include the `data-udi` attribute
